CORPORATION: Add new API to check if player can create corporation (#1598)
bitburner-official/bitburner-src: markdown/bitburner.corporation.createcorporation.md

## Corporation.createCorporation() method

> Create a Corporation. You should use [canCreateCorporation](./bitburner.corporation.cancreatecorporation.md) to check if you can do it before using this function, because it throws an error in some cases.

Reviewer: Not everyone will need to check, but it's good to mention the option.

```suggestion
Create a Corporation. You should use [canCreateCorporation](./bitburner.corporation.cancreatecorporation.md) to check if you are unsure you can do it, because it throws an error in some cases.
```

RAM cost: 20 GB

> This function throws an error if:

Reviewer: Please leave this, it's good/important info. You could probably take out the bit about getBitNodeMultipliers now if you want.

Author: When I put it back, I also fix it. The old "Try to self-fund outside BitNode 3." is wrong. It's "use seed money", not "self-fund". I made that mistake in #1587.
